Analysis/src/Clone.cpp

```diff
@@ -209,8 +210,7 @@ class TypeCloner2
                 p.deprecatedSuggestion,
                 p.location,
                 p.tags,
-                p.documentationSymbol,
-            };
+                p.documentationSymbol, p.typeLocation};
```

Comment:
what's up with this formatting?

Reply:
interesting, not sure what clion did there. Maybe it's because of lack of trailing comma. Will fix